sstables/sstable_directory.cc

```diff
@@ -242,7 +239,7 @@ sstable_directory::process_descriptor(sstables::entry_descriptor desc,
         _max_version_seen = desc.version;
     }
 
-    auto sst = co_await load_sstable(desc, std::move(get_storage_options), flags.sstable_open_config);
+    auto sst = co_await load_sstable(desc, std::invoke(get_storage_options), flags.sstable_open_config);
```

Comment:
Why not just invoke with `()` as usual?
I know this is equivalent, but still it is unexpected.

Reply:
Sure. I have updated the code.